platform/administer/templates/versioning.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: Versioning Templates
+sidebar_label: Versioning
+sidebar_position: 4
+---
+
+import Flow, { Step } from "@site/src/components/Flow";
+import NavStep from "@site/src/components/NavStep";
+import Button from "@site/src/components/Button";
+import Label from "@site/src/components/Label";
+
+Templates can be versioned. Versioning a template allows you
+to make changes over time to a template while retaining the current version/iteration. Resources (for example virtual clusters, namespaces) that are deployed with templates that have updated versions show up with a warning
+indicating that a newer template version is available.
+
+## Template version strings
+
+When creating a virtual cluster from a template, either through the UI or CLI, you can specify a template version string. If omitted, the latest available version is selected by default.
```

Comment:
I would remove “ either through the UI or CLI,” since you leave out other ways (`VirtualClusterInstance` CRD via kubectl or any other K8s tooling) and doesn’t add much here.